packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -143,7 +144,9 @@ export type CallAdapterCallEndedEvent = {
 // @public
 export interface CallAdapterCallManagement {
     // @beta
-    addParticipant(participant: CommunicationIdentifier, options?: AddPhoneNumberOptions): Promise<void>;
+    addParticipant(participant: PhoneNumberIdentifier, options?: AddPhoneNumberOptions): Promise<void>;
+    // (undocumented)
+    addParticipant(participant: CommunicationUserIdentifier): Promise<void>;
```

Comment:
out of interest what happens in a group call here? does it call the participant and show them as calling in the video gallery?

Reply:
I believe if Contoso were to call `adapter.addParticipant(...)` it would fail since the type of call is incorrect for these handlers.